_posts/2020-03-02-using-http-bridge-as-a-kubernetes-sidecar.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title:  "Using HTTP Bridge as a Kubernetes sidecar"
+date: 2020-03-01
+author: jakub_scholz
```

Comment:
What about this blog post? I think you pushed by mistake.

Reply:
Oops. Thanks for noticing.